tests/Integration/Behaviour/Features/Scenario/Product/Combination/update_combination_stock.feature

```diff
@@ -169,4 +169,56 @@ Feature: Update product combination stock information in Back Office (BO)
       | Puff       | Daddy     | 100            |
     And combination "product1SBlack" last stock movement increased by 1
     And product "product1" should have following stock information:
-      | quantity            | 50       |
+      | quantity | 50 |
+
+  Scenario: I update combination stock using fixed quantity
+    Given combination "product1SBlack" should have following stock details:
+      | combination stock detail   | value |
+      | quantity                   | 0     |
+      | minimal quantity           | 1     |
+      | low stock threshold        | 0     |
+      | low stock alert is enabled | false |
+      | location                   |       |
+      | available date             |       |
+    When I update combination "product1SBlack" stock with following details:
+      | fixed quantity | 10 |
+    Then combination "product1SBlack" should have following stock details:
+      | combination stock detail   | value |
+      | quantity                   | 10    |
+      | minimal quantity           | 1     |
+      | low stock threshold        | 0     |
+      | low stock alert is enabled | false |
+      | location                   |       |
+      | available date             |       |
+    When I update combination "product1SBlack" stock with following details:
```

Comment:
You should add some tests regarding the stock movements history, we want to be sure that updating by fixed quantity has the same behaviour regarding the stock movement